src/main/webapp/assets/content/json/irc.json

```diff
@@ -351,6 +351,78 @@
       "linkedin": "charininanayakkara"
     }
   },
+  {
+    "name": "Sanjanee Waniganeththi",
+    "description": "Nuclear Physics",
+    "image": "sanajee.jpg",
```

Comment:
Rename this image to sanjanee.jpg

Reply:
My apologies, I should have noticed that mistake..